doozerlib/distgit.py

```diff
@@ -40,6 +40,8 @@
 ---
 """
 
+BASE_IGNORE = [".git", ".oit", ".gitignore", "additional-tags"]
```

Comment:
Comment for this plz: context as to why this variable exists

Reply:
yes

Reply:
although i might say that we're "ignoring" them in the sense that they don't get cleared out of distgit on rebase, not a rebase ignores source content.